heft-plugins/heft-jest-plugin/src/transformers/StringMockTransformer.ts

```diff
@@ -11,9 +13,12 @@ import type { SyncTransformer, TransformedSource, TransformOptions } from '@jest
  */
 export class StringMockTransformer implements SyncTransformer {
   public process(sourceText: string, sourcePath: string, options: TransformOptions): TransformedSource {
-    // For a file called "myImage.png", this will generate a JS module that exports the literal string "myImage.png"
+    const relativePath: string = relative(options.config.cwd, sourcePath);
```

Comment:
What is `options.config.cwd` here?  Seems like the path should be relative to the project folder that contains **package.json**, but that is not usually called `cwd`.